mail_activity_team/models/mail_activity.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2018 Eficent Business and IT Consulting Services, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import api, models, fields, _
+from odoo.exceptions import ValidationError
+
+
+class MailActivity(models.Model):
+    _inherit = "mail.activity"
+
+    def _get_default_team_id(self):
+        res_model = self.env.context.get('default_res_model', False)
```

Comment:
does it matter if you get `False` or `None` here?

Reply:
what do you mean?

Reply:
`dict.get` returns `None` if the key doesn't exist and you didn't pass another default. So to rephrase: What's the `, False)` part good for?